ec2_metadata.py

```diff
@@ -83,5 +83,10 @@ def region(self):
     def reservation_id(self):
         return requests.get(METADATA_URL + 'reservation-id').text
 
+    @cached_property
+    def security_groups(self):
+        # since this can be one or more items, always return as an array.
+        return requests.get(METADATA_URL + 'security-groups').text.splitlines()
```

Comment:
I don't see anything indicating the order coming from EC2, but I'd rather leave the order alone and let a downstream user decide if they want it sorted. Thoughts?